### PR TITLE
DOC: Style format

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [x] ref #30054
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by jreback** [comment]
cc @datapythonista

File: pandas/core/sorting.py [hunk 1]
```diff
@@ -31,19 +31,24 @@ def get_group_index(labels, shape, sort: bool, xnull: bool):
...
     Parameters
     ----------
-    labels: sequence of arrays
+    labels : sequence of arrays
         Integers identifying levels at each location
-    shape: sequence of ints same length as labels
+    shape : sequence of ints
         Number of unique levels at each location
-    sort: boolean
+    sort : boolean
```
Comment (datapythonista) [inline pandas/core/sorting.py]: bool

File: pandas/core/sorting.py [hunk 2]
```diff
@@ -31,19 +31,24 @@ def get_group_index(labels, shape, sort: bool, xnull: bool):
...
         If the ranks of returned ids should match lexical ranks of labels
-    xnull: boolean
+    xnull : boolean
```
Comment (datapythonista) [inline pandas/core/sorting.py]: bool

File: pandas/core/sorting.py [hunk 3]
```diff
@@ -237,9 +244,21 @@ def nargsort(
...
+    Parameters
+
+    ----------
+
+    kind : str, default 'quicksort'
+
+    ascending : bool, default True
+
+    na_position : {'first', 'last'}, default 'last'
     """
```
Comment (datapythonista) [inline pandas/core/sorting.py]: all these blank lines should be removed, starting at `Parameters` there shouldn't be any

File: pandas/core/sorting.py [hunk 4]
```diff
@@ -303,7 +322,12 @@ def get_flattened_iterator(comp_ids, ngroups, levels, labels):
 
 
 def get_indexer_dict(label_list, keys):
-    """ return a dict of {labels} -> {indexers} """
+    """
+    Returns
+    -------
+    dict
+    labels mapped to indexers.
```
Comment (datapythonista) [inline pandas/core/sorting.py]: Indented and should start with capital `L`

**Review by datapythonista — COMMENTED** [review]
Thanks for the fixes. Few comments but looks good, nice clean up.

Feel free to fix any docstring you want, but the ones of public objects are the ones we try to prioritize, since besides reading them on the code are rendered into the docs.

**Review by datapythonista — APPROVED** [review]
lgtm, thanks for the fixes!

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @MomIsBestFriend

**Pull request merged**